site/content/en/contribute/_index.md

```diff
@@ -1,23 +1,8 @@
-
 ---
 title: "Contribute"
 linkTitle: "Contribute"
+type: docs
 menu:
   main:
     weight: 30
 ---
-
-{{% pageinfo %}}
-This is a placeholder page that shows you how to use this template site.
-{{% /pageinfo %}}
-
-
-This section is where the user documentation for your project lives - all the information your users need to understand and successfully use your project. 
-
-For large documentation sets we recommend adding content under the headings in this section, though if some or all of them don’t apply to your project feel free to remove them or add your own. You can see an example of a smaller Docsy documentation site in the [Docsy User Guide](https://docsy.dev/docs/), which lives in the [Docsy theme repo](https://github.com/google/docsy/tree/master/userguide) if you'd like to copy its docs section. 
-
-Other content such as marketing material, case studies, and community updates should live in the [About](/about/) and [Community](/community/) pages.
-
-Find out how to use the Docsy theme in the [Docsy User Guide](https://docsy.dev/docs/). You can learn more about how to organize your documentation (and how we organized this site) in [Organizing Your Content](https://docsy.dev/docs/best-practices/organizing-content/).
-
```

Comment:
I removed this placeholder text. Is it ok?

Reply:
It looks like this is the page that will describe to users how they can contribute to the site.

I would prefer to keep the placeholder text, until we replace the placeholder text with helpful contributor guides. You're welcome to add that contributor guide in this PR if you'd like, or keep the placeholder text (maybe adding a TODO or WIP to the top).

Reply:
Ah actually, I see you've written up contribution guidelines somewhere else. In that case, I think this is fine to remove.